runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -3187,21 +3143,11 @@ bls12381_p2_decompress_base + bls12381_p2_decompress_element * num_elements`
                         .gas_counter
                         .pay_per(storage_large_read_overhead_byte, read_len as u64)?;
                 }
-                Some(read.deref()?)
+                Some(read.deref(&mut FreeGasCounter)?)
```

Comment:
I'm somewhat confused why this is not participating in TTN-related calculations, but the goal here is to maintain the functional equivalence, so :shrug: